integration-core/src/test/java/org/symphonyoss/integration/core/bridge/StreamServiceTest.java

```diff
@@ -99,6 +101,20 @@ public void init() {
   }
 
   @Test
+  public void testInit() {
+    streamService.init();
+
+    Map<MessageMLVersion, MessageApiClient> apiResolver =
```

Comment:
You don't need to get the internal field to validate this behavior.

You can do it using apiResolver field:
```
apiResolver.clear();
streamService.init();
```